userguide/drawing_classifier/how-it-works.md

```diff
@@ -26,7 +26,7 @@ example_drawing = [
 ]
 ```
 
-When stroke-based drawing data is given as input to `turicreate.drawing_classifier.create`, a series of pre-processing steps are applied to convert the stroke-based drawings to bitmaps which is then consumed by the neural network. This data transformation can be visualized through the `turicreate.drawing_classifier.util.draw_strokes(...)` Python API as well. 
+When stroke-based drawing data is given as input to `turicreate.drawing_classifier.create`, a series of pre-processing steps are applied to convert the stroke-based drawings to bitmaps which is then consumed by the neural network. This data transformation can be visualized through the `turicreate.drawing_classifier.util.draw_strokes(...)` Python API as well. It is a known issue with models trained on Linux stroke data not generalizing well to drawings rendered on Macs and vice-versa.
```

Comment:
Why are we adding this to the user guide as opposed to opening an issue about it? It's going to be hard to remember to come back and update this when we fix it.

Reply:
Also, it's not clear what trained on "Linux stroke data" means. Models trained on Linux using stroke data collected from Macs?

Reply:
We do have an issue about it. We will not be able to resolve it for our present release. It's not obvious from usage of the toolkit that models trained from stroke data on Linux seem not to generalize well to Mac.

Reply:
It is an open issue. (refer to #1956)
I referred the PR here so we remember to update the guide. The point according to me was to not let users have a bad experience since it a known issue that cannot be fixed yet. 
It doesn't matter where data was collected. It is the stroke data converted to bitmap done while training in Linux which will not generalize well on stroke data converted to bitmap in Mac.

Reply:
Hmm, maybe it would be clearer to flip this and suggest how to maximize the likelihood of good generalization. Something along the lines of "It is a known issue that the model may be sensitive to platform-specific rendering from strokes. For the best generalization performance, train on the same platform where the model will be deployed."
?

Reply:
@nickjong That does sound better.